adobe_vipm/flows/fulfillment/shared.py

```diff
@@ -611,7 +611,11 @@ def coterm_and_next_sync_update_if_needed(self, context, coterm_date, next_sync)
     def commitment_update_if_needed(self, context):
         if not context.adobe_customer:
             return False
-        commitment = get_3yc_commitment_request(context.adobe_customer)
+        commitment = (
+            get_3yc_commitment_request(context.adobe_customer)
```

Comment:
Just to confirm, `get_3yc_commitment_request` has `is_recommitment` param. Is it OK to use here only the case with its default value which is `False`?